src/main/java/de/tum/cit/aet/artemis/iris/service/pyris/PyrisWebhookService.java

```diff
@@ -260,17 +261,17 @@ public void autoUpdateFaqInPyris(Long courseId, Faq newFaq) {
         IrisCourseSettings courseSettings = irisSettingsRepository.findCourseSettings(courseId).isPresent() ? irisSettingsRepository.findCourseSettings(courseId).get() : null;
         if (courseSettings != null && courseSettings.getIrisFaqIngestionSettings() != null && courseSettings.getIrisFaqIngestionSettings().isEnabled()
                 && courseSettings.getIrisFaqIngestionSettings().getAutoIngestOnFaqCreation()) {
-            addFaqToPyris(newFaq);
+            addFaq(newFaq);
         }
     }
 
     /**
-     * adds the faq to the vector database in Pyris
+     * adds the faq to Pyris. It is triggered when the user manually sends the faqs via the button in the UI
```

Comment:
> It is triggered when the user manually sends the faqs via the button in the UI

I wouldn't add this here. The server does not depend on the client by design of a client-server architecture. I think we can just remove it